### PR TITLE
Feature/sections

### PR DESCRIPTION
**Pull request by pananimals**

### CODE REVIEW

**Comment by michalkowol** [comment]
```
ERROR in 
    src/css/main.scss
     36:2  ✖  Unexpected whitespace at end of line   no-eol-whitespace
```

- [ ] remove whitespace at end of line

File: src/css/styles/_sections.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+.section__p--3 {
+  margin-bottom: 0;
+}
+
+.section__list {
+  padding: 0;
+  list-style: none;
+  counter-reset: list-counter;
+}
```
Comment (michalkowol) [inline src/css/styles/_sections.scss]: IMHO list is a separate block and this should not be connected with section block

- [ ] list as separate block

File: src/css/styles/_sections.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+.section__list-item__inner {
+  position: absolute;
+  bottom: 1rem;
+  left: 5rem;
+  padding-right: 1rem;
+}
+
+// PRODUKCJA
```
Comment (michalkowol) [inline src/css/styles/_sections.scss]: - [ ] remove `// PRODUKCJA`

File: src/js/adjustHeights.js [hunk 3]
```diff
@@ -1,27 +1,42 @@
...
 
 // adjust height of a div to the highest child of all considered divs
-export function setHeights() {
-  setHeightsArray.forEach((arrayEl) => {
-    const elements = $(`#${arrayEl.parentId}`).find(`.${arrayEl.className}`).toArray();
-    const heights = elements.map(it => it.scrollHeight);
-    const maxHeight = _.max(heights);
-    elements.forEach((el) => { el.style.height = `${maxHeight}px`; });
-  });
+function setHeightsInner(array) {
```
Comment (michalkowol) [inline src/js/adjustHeights.js]: rename `array` to `elementsWithParents`
and `arrayEl` to `elementWithParent`

- [ ] rename `array`

File: src/js/adjustHeights.js [hunk 4]
```diff
@@ -1,27 +1,42 @@
...
-  // { className: 'flex-col--2__list', parentId: 'naprawa'},
+const setHeightsArrayDesktop = [
+  { className: 'section__header', parentId: 'naprawa' },
+  { className: 'section__p--1', parentId: 'naprawa' },
+  { className: 'section__p--2', parentId: 'naprawa' },
+  { className: 'section__list-div', parentId: 'naprawa' },
+  { className: 'section__header', parentId: 'produkcja' },
+  { className: 'section__p--1', parentId: 'produkcja' },
+  { className: 'section__p--2', parentId: 'produkcja' },
+  { className: 'section__list-div', parentId: 'produkcja' }
```
Comment (michalkowol) [inline src/js/adjustHeights.js]: Please change name of `className` to `element` and `parentId` to `parent`.
I see some inconsistency - once you use `class` and then you use `id` to refer to css class of element

- [ ] inconsistency of naming

File: src/js/adjustHeights.js [hunk 5]
```diff
@@ -1,27 +1,42 @@
...
-  });
+function setHeightsInner(array) {
+  return () => {
+    array.forEach((arrayEl) => {
+      const elements = $(`#${arrayEl.parentId}`).find(`.${arrayEl.className}`).toArray();
+      const heights = elements.map(it => it.children[0].scrollHeight);
+      const maxHeight = _.max(heights);
+      elements.forEach((el) => { el.style.height = `${maxHeight}px`; });
+    });
+  };
```
Comment (michalkowol) [inline src/js/adjustHeights.js]: Just a thought - maybe we should introduce `.js-eq-height-container` and `.js-eq-height`, mark elements in html and run this script. IMHO we could get rid of `setHeightsArrayDesktop`

File: src/js/adjustHeights.js (hunk 4 above)
Comment (michalkowol) [reply src/js/adjustHeights.js]: >  I can rename className to elementClass to be more consistent, though.
Great!

**Comment by michalkowol** [comment]
I talked with Aurator and it turned out that there will be list. It will be larger (two columns wide).

**Comment by pananimals** [comment]
Ok, I added more space between columns' texts, but between columns I think I already put as much space as possible For narrow desktops long texts in ordered list are an issue- I tried to allocate as much width as possible for them, but they still get pretty "tall" at times. I may try to introduce percent-wise spacing if necessary.
Btw., when you post a comment like the one with column spacing, I'm not able to comment it back- hence the new general comment.

**Pull request merged**